Firestore/core/test/firebase/firestore/util/executor_std_test.cc

```diff
@@ -70,7 +70,6 @@ TEST_F(ScheduleTest, PopIfDue_Delayed) {
   schedule.Push(2, start_time + chr::milliseconds(3));
   schedule.Push(3, start_time + chr::milliseconds(1));
 
-  EXPECT_FALSE(schedule.PopIfDue().has_value());
```

Comment:
If there was a delay between this line and last push, there might already be a value due.